.github/scripts/simpleauth-azdo-pipelines.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+countNum=1
+restUrl="https://dev.azure.com/mseng/VSIoT/_apis/build/latest/$2?api-version=6.0-preview.1"
+rsp=$(curl -u :$1 $restUrl | jq -r '.value| .[0]')
+status=$(echo $rsp | jq -r '.state')
+buildId=$(echo $rsp | jq -r '.id')
+echo "===== build id is: " $buildId
+echo "===== build pipeline status: " $status
+while [[ $countNum -le 50 && "$status" != "completed" ]]
+do 
+    sleep 1m
+    rsp=$(curl -u :$1 $restUrl | jq -r '.value| .[0]')
+    status=$(echo $rsp | jq -r '.state')
+    echo "loop status" $status
+    countNum=$(( $countNum + 1 ))
```

Comment:
in loop, every time sleep 1m, and fetch pipeline status, if status is completed or exceed 50 times, break